md2html/md2html.c

```diff
@@ -288,6 +293,7 @@ cmdline_callback(int opt, char const* value, void* data)
 
         case 'c':   parser_flags = MD_DIALECT_COMMONMARK; break;
         case 'g':   parser_flags = MD_DIALECT_GITHUB; break;
+        case 'R':   parser_flags = MD_DIALECT_REDDITPOST; break;
 
```

Comment:
It's a minor detail, but please swap 'r' and 'R' so that there is some consistency: Lower case for dialect options, upper case for extensions.